pkg/providers/nutanix/provider.go

```diff
@@ -607,8 +607,8 @@ func (p *Provider) InstallCustomProviderComponents(ctx context.Context, kubeconf
 }
 
 func (p *Provider) PreCAPIInstallOnBootstrap(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error {
-	// TODO(nutanix): figure out if we need something else here
-	return nil
+	logger.V(0).Info("Installing secrets on bootstrap cluster")
```

Comment:
nit: you don't need to specify verbosity if it's 0
```suggestion
	logger.Info("Installing secrets on bootstrap cluster")
```

Reply:
Cool, fixed.